packages/backend/src/ee/services/ai/agents/tests/testUtils.ts

```diff
@@ -0,0 +1,424 @@
+import {
+    CompiledDimension,
+    CompiledMetric,
+    DimensionType,
+    Explore,
+    FieldType,
+    MetricType,
+} from '@lightdash/common';
+import { getOpenaiGptmodel } from '../../models/openai-gpt';
+import type { AiAgentArgs, AiAgentDependencies } from '../../types/aiAgent';
+import { AiAgentExploreSummary } from '../../types/aiAgentExploreSummary';
+
+export type ConversationHistory = Array<{
+    role: 'user' | 'assistant';
+    content: string;
+}>;
+
+export const createMessage = (
+    content: string,
+    role: 'user' | 'assistant' = 'user',
+) => ({
+    role,
+    content,
+});
+
+const capitalize = (str: string): string =>
+    str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
what do we use in actual code?

Reply:
another capitalize, I'll update to use that!